husky_components/datagrid/main.js

```diff
@@ -18,6 +18,7 @@
  * @param {String} [options.columnOptionsInstanceName] if set, a listener will be set for listening for column changes
  * @param {String} [options.url] url to fetch data from
  * @param {String} [options.instanceName] name of the datagrid instance
+ * @param {Array} [options.expandIds] array of ids which is appended to expandIds-parameter when loading list items
```

Comment:
Should we really introduce a new option here?  Should simply the preselected elements be expanded? /cc @chirimoya @wachterjohannes